.github/workflows/pr.yml

```diff
@@ -0,0 +1,35 @@
+name: Pull Request
+
+# Run on PRs.
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
```

Comment:
the Pipfile requires at least `3.7` so we can remove version lower than that right? We can definitely remove `2.7`

Reply:
I don't think we use anything specific to Python3 so I'd like to support `2.7` as well. Lots of people are still on the old version and it seems easy with CI and since this is a small library to keep it supported.